.github/workflows/bImage.yml

```diff
@@ -0,0 +1,35 @@
+name: "bImage"
+on:
+  push:
+    branches:
+      - main
+      - dev
+      - githactions
```

Comment:
This line is not necessary when merging.

Reply:
Thank you for catching that. I added for the test. I will remove that.